asyncio_apns/apns_connection.py

```diff
@@ -82,15 +82,18 @@ def _prepare_request(self, payload: Union[Payload, str], token: str,
         ]
         if topic:
             request_headers.append(('apns-topic', topic))
+        if extra_headers:
+            for h_tuple in extra_headers:
+                request_headers.append(h_tuple)
```

Comment:
`request_headers.extend(extra_headers)` does the same in single line :)

Reply:
Quite so!